File: youlexuan-shop-web/src/main/webapp/js/service/userService.js
app.service('userService',function ($http) {
    this.delete = function (id) {
        return $http.get('/usercenter/delete.do?ids='+ id);
    };
    this.findPage = function (pageNum,pageSize,entity) {
        return $http.post('../usercenter/findPage.do?pageNum='+pageNum+'&pageSize='+pageSize,entity);
    }
})

File: youlexuan-usercenter-interface/src/main/java/com/offcn/usercenter/service/UserService.java
package com.offcn.usercenter.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbUser;

public interface UserService {
    /**
     * 带有条件的分页查询
     * @param pageNum
     * @param pageSize
     * @param tbUser
     * @return
     */
    public PageResult findPage(int pageNum, int pageSize, TbUser tbUser);


    /**
     * 批量删除
     * @param ids
     */
    public void delete(Long[] ids);
}
